apps/api/src/app/tenant/tenant.controller.ts

```diff
@@ -51,6 +54,8 @@ import { UserAuthentication } from '../shared/framework/swagger/api.key.security
 
 import { SdkUsePagination } from '../shared/framework/swagger/sdk.decorators';
 
+const v2Description = ' Tenants is not supported in code first version of the API.';
```

Comment:
As below, naming this semantically.
```suggestion
const v2TenantsApiDescription = ' Tenants is not supported in code first version of the API.';
```